src/main/java/twistedgate/immersiveposts/common/blocks/HorizontalTrussBlock.java

```diff
@@ -36,6 +36,7 @@
 import net.minecraft.world.phys.shapes.VoxelShape;
 import net.minecraftforge.api.distmarker.Dist;
 import net.minecraftforge.api.distmarker.OnlyIn;
+import org.jetbrains.annotations.NotNull;
```

Comment:
Never seen that one before.

i tend to just use `javax.annotation.Nonnull;`

Reply:
There's no difference

Reply:
There is a difference, and one you're not recognizing: One is a JetBrains annotation, and not everyone uses JetBrains products.